src/pages/index.tsx

```diff
@@ -1,19 +1,126 @@
-        width={500}
-        height={500}
-      />
+    <Flex direction="column" h="calc(100vh - 100px)">
+      <Stack
+        justifyContent="center"
+        alignItems="center"
+        direction={{ base: "column-reverse", md: "row" }}
+        marginBottom="10"
+        flex="1"
+        flexBasis="250px"
+      >
+        <Box bg="white" w={{ md: "50%" }} p={10}>
+          <Heading
+            fontSize={{ base: "x-large", md: "xx-large", lg: "xxx-large" }}
+            marginBottom="1"
+          >
+            Non-profit Portal
+          </Heading>
+          <Text fontSize="medium" marginY="5" maxW={500}>
+            We build software, websites, and mobile apps to turn your ideas into
+            into real products. If you have an existing project or a happy to
+            get involved in your development cycle.
+          </Text>
+          <Button colorScheme="blue">Log In</Button>
+        </Box>
+        <Image
+          src={FirstLandingImage}
+          alt="Build Websites"
+          width={250}
+          height={250}
+        />
+      </Stack>
+      <Stack
+        justifyContent="center"
+        alignItems="center"
+        direction={{ base: "column", md: "row" }}
+        marginBottom="10"
+        flex="1"
+      >
+        <Image
+          src={SecondLandingImage}
+          alt="Build Websites"
+          width={190}
+          height={190}
+        />
+        <Box w={{ md: "50%" }} p={10} marginLeft={{ md: "20" }}>
+          <Heading
+            fontSize={{ base: "large", md: "x-large" }}
+            color="rgb(0,105,202)"
+            marginBottom="5"
+          >
+            Start a New Project with us
+          </Heading>
+          <Text fontSize={{ base: "small", md: "medium" }}>
+            Have a great idea for how a website or mobile application can help
+            your nonprofit reach more people? Let us know, and we can build it
+            for you!
+          </Text>
+        </Box>
+      </Stack>
+      <Stack
+        justifyContent="center"
+        alignItems="center"
+        direction={{ base: "column", md: "row" }}
+        flex="1"
+      >
+        <Image
+          src={ThirdLandingImage}
+          alt="Build Websites"
+          width={190}
+          height={190}
+        />
+        <Box w={{ md: "50%" }} p={10}>
+          <Heading
+            fontSize={{ base: "large", md: "x-large" }}
+            color="rgb(0,105,202)"
+            marginBottom="5"
+          >
+            Project Maintenance
+          </Heading>
+          <Text fontSize={{ base: "small", md: "medium" }}>
+            Have a completed product for Bits of Good that requires bug fixes or
+            maintenance? Let us know, and we can fix it for you!
+          </Text>
+        </Box>
+      </Stack>
+      <Box marginX="7" marginBottom="5">
+        <Divider borderColor="grey" />
+      </Box>
+      <HStack justifyContent="center" spacing="10">
+        <Box justifyContent="center">
```

Comment:
Under public/images, there is a small_logo.svg file you can use here in place of the hack4impact text.